app/controllers/admin/official_postcodes_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Admin
+  class OfficialPostcodesController < Admin::ApplicationController
```

Comment:
I am not sure I understand what `Official Postcodes` means.

Can't we just use the `PostcodesController`?